src/_scss/core/_variables.scss

```diff
@@ -92,6 +92,7 @@ $theme-color-3: $blue-warm-70v;
 $theme-color-4: $indigo-cool-80v;
 $theme-color-5: $blue-warm-5v;
 $theme-color-6: $blue-vivid-60v;
+$theme-color-7: #ccecf2;
```

Comment:
can you follow the convention of creating a variable for cyan-10: #ccecf2
then assigning theme-color-7: $cyan-10

Reply:
Good to know. Added